### PR TITLE
Add view property cautionary alert record endpoint

### PR DESCRIPTION
**Pull request by LBHMGeorgieva**
- Added view property cautionary alert(s) record endpoint
- Query based on property reference (based on latest conversation with Nick Prince)

### CODE REVIEW

File: CautionaryAlertsApi.Tests/V1/Infrastructure/ExampleContextTests.cs [hunk 1]
```diff
@@ -21,6 +21,18 @@ public void CanGetADatabaseEntity()
 
             var result = UhContext.PeopleAlerts.ToList().FirstOrDefault();
 
+            result.Should().BeEquivalentTo(databaseEntity);
```
Comment (elcorbs) [inline CautionaryAlertsApi.Tests/V1/Infrastructure/ExampleContextTests.cs:24]: This file needs renaming to `UhContextTests`

File: CautionaryAlertsApi/V1/Boundary/Response/CautionaryAlertsPropertyResponse.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CautionaryAlertsApi.V1.Boundary.Response
+{
+    public class CautionaryAlertsPropertyResponse
+    {
+        public string PropertyReference { get; set; }
+        public string AddressNumber { get; set; }
```
Comment (elcorbs) [inline CautionaryAlertsApi/V1/Boundary/Response/CautionaryAlertsPropertyResponse.cs:11]: Do we not want to return UPRN here as well?

File: CautionaryAlertsApi/V1/Gateways/UhGateway.cs [hunk 3]
```diff
@@ -45,13 +47,51 @@ private List<PersonAlert> GetPersonAlerts(ContactLink link)
...
                 .OrderByDescending(a => a.DateModified)
                 .FirstOrDefault(a => a.AlertCode == alert.AlertCode);
             return alert.ToDomain(description?.Description);
         }
+
+
+        public CautionaryAlertsProperty GetCautionaryAlertsForAProperty(string propertyReference)
+        {
+            var addressLink = _uhContext.Addresses.
+                Where(x => x.PropertyReference == propertyReference).FirstOrDefault();
```
Comment (elcorbs) [inline CautionaryAlertsApi/V1/Gateways/UhGateway.cs:62]: Is there typically just one record in the database for each `propertyReference`? I'm wondering if we want to apply some sort of ordering before we take the first record.

Comment (LBHMGeorgieva) [reply CautionaryAlertsApi/V1/Gateways/UhGateway.cs:62]: I believe it's always one record, I will check the data again.

Comment (LBHMGeorgieva) [reply CautionaryAlertsApi/V1/Gateways/UhGateway.cs:62]: Only less than 10 records have a duplicate prop ref and none of them relating to a cautionary alert so for now I think we can stick with first or default.

File: CautionaryAlertsApi/V1/Boundary/Response/CautionaryAlertsPropertyResponse.cs (hunk 2 above)
Comment (LBHMGeorgieva) [reply CautionaryAlertsApi/V1/Boundary/Response/CautionaryAlertsPropertyResponse.cs:12]: Added UPRN to response

**Pull request merged**